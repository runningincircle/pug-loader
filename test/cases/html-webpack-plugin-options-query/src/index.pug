.title= htmlWebpackPlugin.options.title
.teaser= htmlWebpackPlugin.options.pluginData.teaser
.description= dataFromQuery.options.description
img(src=require('./image.png'))