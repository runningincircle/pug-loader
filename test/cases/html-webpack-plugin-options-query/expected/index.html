<div class="title">Home</div><div class="teaser">Teaser of home page.</div><div class="description">Home custom data.</div><img src="assets/images/image.697ef306.png">